backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -110,6 +111,60 @@ object BackendSettings extends BackendSettingsApi {
     }
   }
 
+  override def supportFileFormatWrite(format: FileFormat, fields: Array[StructField]): Boolean = {
+    // Validate if all types are supported.
+    def validateTypes(fields: Array[StructField]): Boolean = {
+
+      val unsupportedDataTypes =
+        fields.flatMap {
+          field =>
+            field.dataType match {
+              case _: TimestampType => Some("TimestampType")
```

Comment:
@ulysses-you Velox support timestamp write. But the results are inconsistent with Spark, with a time difference of eight hours. It may be due to incorrect timezone settings, which can be addressed in subsequent PR.